packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -854,7 +856,16 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends React.Componen
   };
 
   protected getActiveDescendant() {
+    if (this.state.suggestionsLoading) {
+      return undefined;
+    }
+
     const currentIndex = this.suggestionStore.currentIndex;
+    // if the suggestions element has actions and the currentIndex does not point to a suggestion, return the action id
+    if (currentIndex < 0 && this.suggestionElement.current && this.suggestionElement.current.hasSuggestedAction()) {
+      return 'sug-selectedAction';
```

Comment:
Can this and the other ID values go in a constant instead of being magic values? Also, what will happen if there's more than one picker on the page?

Reply:
@ecraig12345 thanks for the review! I followed the format of the picker item id values when I created this one -- they're also magic values of the format `sug-${index}` (controlled on line 869 immediately below this block).

I can't speak to the original reason that approach was chosen, but I'm guessing it's because the id is shared across the BasePicker and Suggestions components, and using a constant like `this._id` would require an additional property passed into the Suggestions component. On the plus side, multiple pickers per page shouldn't be an issue, since it shouldn't be possible to have more than one open at once.

Let me know if that answers your concern :)

Reply:
That makes sense. Probably not the best design originally (the constants should at least be in a shared file somewhere) but this change doesn't make it significantly worse.